src/app/components/shell/header/header.scss

```diff
@@ -1,12 +1,31 @@
+// scss-lint:disable Indentation, PropertyUnits
+// scss-lint:disable all
+
 @import 'variables';
 @import 'mixins';
 
 .page-header {
     background-color: color(neutral-lightest);
     overflow: hidden;
-    position: relative;
     transition: max-height 0.5s;
     z-index: 1;
+    left: 0;
+    position: fixed;
+    right: 0;
+    top: 0;
+    display: flex;
+    flex-flow: column;
+    width: 100%;
+    height: 70px;
+
+    @include medium-up {
+      height: 120px;
+      position: absolute;
+      top: auto;
+      left: auto;
+      right:auto;
+      overflow: visible;
+    }
```

Comment:
Whitespacing seems to be inconsistent.  There's 4 spaces for indentation beginning with `.page-header`, but 2 spaces in `@include medium-up {`.  Can you make this a consistent 4 spaces (in every file)?